components/provisional-en.html

```diff
@@ -158,13 +158,49 @@ <h3><code>.experimental</code> depricated</h3>
 		<td>
 			<p>New layout and design pattern. To support the theme/topic beta template.</p>
 			<ul>
-				<li><a href="theme-topic-en.html">English working example</a></li>
-				<li><a href="theme-topic-fr.html" hreflang="fr">French working example</a></li>
+				<li><a href="theme-topic/theme-topic-en.html">English working example</a></li>
+				<li><a href="theme-topic/theme-topic-fr.html" hreflang="fr">French working example</a></li>
 			</ul>
 		</td>
 		<td>v9.1.0</td>
 		<td></td>
 	</tr>
+	<tr>
+		<th>Component <code>.gc-most-requested</code></th>
+		<td>
+			<p>This adds a most requested section that features top tasks related to the page it is on.</p>
+			<ul>
+				<li><a href="gc-most-requested/gc-most-requested-en.html">English documentation and working example</a></li>
+				<li><a href="gc-most-requested/gc-most-requested-fr.html" hreflang="fr">French documentation and working example</a></li>
+			</ul>
+		</td>
+		<td>v9.4.0</td>
+		<td></td>
+	</tr>
+	<tr>
```

Comment:
No need to populate this page, this page is deprecated in favour of an self-maintained way through exposing the component state in the index.json-ld (which will be changed soon by a special unpublished front-matter)